.drone.yml

```diff
@@ -267,6 +267,28 @@ steps:
     event:
     - tag
 
+- name: docker-publish-suc
+  image: rancher/drone-images:docker-s390x
+  volumes:
+    - name: docker
+      path: /var/run/docker.sock
+  settings:
+    dockerfile: package/Dockerfile.suc
+    password:
+      from_secret: docker_password
+    repo: "rancher/system-agent"
+    tag: "${DRONE_TAG}-suc"
```

Comment:
This tag will clobber the existing `-suc` tag. 

Do we need to specify this as `-s390x-suc`? Notably, we don't support SUC upgrades on platforms beyond amd64 currently.

Reply:
@Oats87 Fixed it. Added a manifest-suc pipeline.